src/game/Utils/_DutchText.cc

```diff
@@ -2423,8 +2423,8 @@ static const wchar_t *s_dut_BobbyRText[BobbyRText_SIZE] =
 
 
 /* The following line is used on the Ammunition page.  It is used for help text
- * to display how many items the player's merc has that can use this type of
- * ammo. */
+	* to display how many items the player's merc has that can use this type of
+	* ammo. */
```

Comment:
It was like this in many other places (indented with tabs). I would prefer `//` style comments.

If we choose to use spaces, we would have mixed indentation in many places.